src/Azure.Functions.Cli/Helpers/VersionHelper.cs

```diff
@@ -13,11 +17,26 @@ namespace Azure.Functions.Cli.Helpers
 {
     internal class VersionHelper
     {
+        public static async Task<string> RunAsync(Task<(bool, string)> isRunningOlderVersion = null)
+        {
+            isRunningOlderVersion ??= IsRunningAnOlderVersion();
+
+            (var isOlderVersion, var olderVersionWarning) = await isRunningOlderVersion;
+
+            var multipleInstallsWarning = await HaveMultipleInstallations(isOlderVersion);
+
+            if (!string.IsNullOrEmpty(multipleInstallsWarning))
+            {
+                return multipleInstallsWarning;
+            }
+
+            return olderVersionWarning;
+        }
 
         // Check that current core tools is the latest version. 
         // To ensure that it doesn't block other tasks. The HTTP Request timeout is only 1 second. 
         // We simply ingnore the exception if for any reason the check fails. 
-        public static async Task<string> CheckLatestVersionAsync()
+        public static async Task<(bool,string)> IsRunningAnOlderVersion()
```

Comment:
Can we just make this function return Task<bool>? The only place where that message seems to be used anyways is for L33 of this file, so couldn't we just replace `olderVersionWarning` with `Constants.OldCoreToolsVersionMessage`?

Reply:
Addressing it.

Reply:
I have addressed it.